docs/book/deployment.md

```diff
@@ -309,6 +309,51 @@ ignore any file types it does not recognize. These files must have the file exte
 `.png`, `.jpeg`, `.jpg`, `.mp3`, `.json`, or`.txt`. Litexa extensions may add to that list.
 Please see the [section on assets](/book/presentation.html#asset-file-references) for more information.
 
+:::tip Overriding a deployment's assets root path
```

Comment:
minor, for consistency with other documentation: `deployment's` -> `deployment target's`